web/src/message_edit.js

```diff
@@ -1232,6 +1248,7 @@ export function delete_topic(stream_id, topic_name, failures = 0) {
 }
 
 export function handle_narrow_deactivated() {
+    assert(message_lists.current !== undefined);
     for (const [idx, elem] of currently_editing_messages) {
         if (message_lists.current.get(idx) !== undefined) {
             const $row = message_lists.current.get_row(idx);
```

Comment:
This would be a follow-up, but I think this function might be a duplicate of the logic calling `message_edit.maybe_show_edit` in `message_list_view.js`.

Reply:
It is required to call it since `_post_process_single_row` is not called when narrow is deactivated.

Reply:
OK, I see ... or really I guess most correctly, it's not called when showing an already pre-rendered view.